docs/OV_Runtime_UG/Samples_Overview.md

```diff
@@ -265,9 +271,9 @@ PATH=<INSTALL_DIR>\runtime\bin;%PATH%
 ```
 where `<INSTALL_DIR>` is the directory in which the OpenVINO toolkit is installed.
 
-You are ready to run sample applications. To learn about how to run a particular
+Sample applications is now ready to be start. To learn about how to run a particular
 sample, read the sample documentation by clicking the sample name in the samples
```

Comment:
sample, go to the documentation by clicking the name of a chosen sample from the list above.